### ISSUE TITLE
[Feature] Make port definitions on public LB subnet more restrictive

### ISSUE THREAD

**Issue by denismakogon** [oracle-terraform-modules/terraform-oci-oke#391]
For the following subnet:
- public LB

security list allows traffic to restricted ports
```
Direction: ingress
Protocol: TCP
Ports to close: 11, 17 - 19, 21, 23 - 25, 43, 49, 53, 70 - 74, 79 - 81, 88, 111, 123, 389, 445, 500, 636, 3306, 3389, 5901, 5985 - 5986, 7001, 8000, 8080, 8443, 8888
```

This has to be addressed by introducing more tight security list rules definitions to prevent access to a set of restricted ports.

**Comment by hyder** [comment]
Have you tried using public_lb_ports to restrict the ports opened on the public load balancer? By default, we open only ports 80 and 443.

**Comment by denismakogon** [comment]
Wish it accepted port ranges as well as single port.

So, I tried, it doesn't serve my needs.

**Comment by hyder** [comment]
Does specifying the list of ports in current format actually not work or is it because you have too many ports to open and specifying individual ports result in a long list and is not practical? If it's the 1st case, then we have a bug. If it's the 2nd case, you want an enhancement to provide a list of allowed ports. Which one is it?

**Comment by denismakogon** [comment]
> Does specifying the list of ports in the current format actually not work

because it accepts only a single port, but not port ranges, so, instead of `[80, 443]`, it must accept things like this: `[80, "9001-9010", 443]`

**Comment by hyder** [comment]
Your answer is confusing. Does it work if you specify the list of single ports even if it's a long list or does it still not work?

**Comment by denismakogon** [comment]
It doesn't work, period. The reason why it doesn't work is in the implementation.

**Comment by hyder** [comment]
Closed in #396
